ci/scripts/openstack/utils.sh

```diff
@@ -583,3 +583,53 @@ delete_sg() {
 
   openstack security group delete "${SG}" > /dev/null 2>&1 || true
 }
+
+# Description:
+# Shares image beween projects.
+#
+# Usage share_image <IMAGE_ID> <PROJECT_ID>
+share_image() {
+  local PROJECT_ID IMAGE_ID
+
+  IMAGE_ID="${1:?}"
+  PROJECT_ID="${2:?}"
+
+  openstack image set --shared "${IMAGE_ID}"
+  openstack image add project "${IMAGE_ID}" "${PROJECT_ID}"
+}
+
+# Description:
+# Accepts shared image.
+#
+# Usage accept_shared_image <IMAGE_ID>
+accept_shared_image() {
+  local IMAGE_ID
+  IMAGE_ID="${1:?}"
+  openstack image set --accept "${IMAGE_ID}"
+}
+
+# Description:
+# Gets Project ID by Project Name.
+#
+# Usage get_project_id <PROJECT_NAME>
+
+get_project_id() {
+  local PROJECT_NAME PROJECT_INFO
+
+  PROJECT_NAME="${1:?}"
+  PROJECT_INFO="$(openstack project list  -f json)"
+  PROJECT_ID="$(echo "${PROJECT_INFO}" |  jq -r --arg projectName "${PROJECT_NAME}" '.[] | select(.Name == $projectName ).ID')"
```

Comment:
I think it would be better echo the project ID at the end instead of setting the variable. Then the user can decide what variable name to use.
```suggestion
  echo "${PROJECT_INFO}" |  jq -r --arg projectName "${PROJECT_NAME}" '.[] | select(.Name == $projectName ).ID'
```

Reply:
fixed